examples/python/sqlc.json

```diff
@@ -10,7 +10,8 @@
           "out": "src/authors",
           "package": "authors",
           "emit_sync_querier": true,
-          "emit_async_querier": true
+          "emit_async_querier": true,
+          "use_pydantic_models": true
```

Comment:
Instead of updating the example, please add a new end-to-end test for Pydantic models.